saas/models.py

```diff
@@ -3458,6 +3460,24 @@ def new_payable(customer, price, provider, descr,
             orig_account=Transaction.RECEIVABLE,
             orig_organization=provider)
 
+    @staticmethod
+    def new_receivable(customer, price, provider, descr,
```

Comment:
When I created this I had little knowledge about accounting, so I thought that if I just replace the orig_* values with dest_* values that would create a transaction which reverses the payable, and results in a receivable. Finally, receivable, like payable, seems to be a common use case in accounting. Does this method produce wrong transactions or it doesn't make sense from code perspective (not enough utility to create a separate method, etc)?

Reply:
Reversing the `Transaction` is a good idea (and that's how it would be done, correct). I see how this method is used in `renewals.py`.
The problem with reversing `Transaction`s is that the intermediate balance (before the end of period when `renewals` is called) will be inflated. Even if it is a bit more compute intensive on each use charge event, I believe checking for over-quota on  `record_use_charge` will generate less bugs moving forward.